### ISSUE TITLE
User interface: "help" inside "settings"

### ISSUE THREAD

**Issue by jpellegrini** [souch/SMP#32]
I was commenting on bug #29 when I realized that Sicmu has a "help" page inside the "settings" page...
This doesn't look like the most natural location for the help page. (Most people, I suppose, will not get into "settings" in order to find out how to use the app).

Maybe it would be better to move the help page somewhere else?

**Comment by souch** [comment]
Yep, but no :-). I should just put a warning at first smp startup that help is in settings. Anyway, nobody reads helps page :!
